src/cc/lib/graph/metadata.h

```diff
@@ -10,13 +10,17 @@
 
 namespace snark
 {
+
+const size_t MINIMUM_SUPPORTED_VERSION = 1;
```

Comment:
Can we avoid duplicate magic numbers or otherwise link to eachother? Minimum_supported_version should always be the same as the version right, since there is no need to update the version otherwise?

Reply:
* It is hard to use a single value, one is in C++ runtime value, another is in Python runtime.
* Changes don’t have to be breaking, some might be incremental.